src/components/faq.tsx

```diff
@@ -84,3 +60,254 @@ const AccordianMenu = ({
     </div>
   );
 };
+
+export const faqs: FAQ[] = [
+  {
+    id: "what-mento",
+    question: "What is the Mento Protocol?",
+    answer: (
+      <span>
+        <Link
+          className="font-fg text-primary-blue underline text-sm"
+          href="https://www.mento.org/"
```

Comment:
Might want to consider dark & mobile (unsure if needed or what implementation is here)

Reply:
This is fine on mobile, will leave dark mode for another issue as it will blow this issue up.